modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java

```diff
@@ -124,6 +124,7 @@ public class CodegenModel implements IJsonSchemaValidationProperties {
     @Getter @Setter
     public String arrayModelType;
     public boolean isAlias; // Is this effectively an alias of another simple type
```

Comment:
FYI. isAlias was created a while ago mainly for top level schema using $ref to primitive type.

We plan to remove it in the future.

If the rust server generator needs to use it somehow, please share the details of the use cases.